code/modules/client/preferences.dm

```diff
@@ -1979,6 +1979,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 	character.dna.features = features.Copy()
 	character.set_species(chosen_species, icon_update = FALSE, pref_load = TRUE)
+	character.dna.features = features.Copy()
```

Comment:
Would be best to have MonkeStation Edit comments for non-modular files, just saying that we changed this locally,
catch any issues with upstream merges.
```suggestion
	character.dna.features = features.Copy() //MonkeStation Edit: Character generation bugfix
```